pkgs/development/python-modules/readme_renderer/default.nix

```
@@ -1,40 +1,45 @@
 { lib
+, bleach
 , buildPythonPackage
-, fetchPypi
-, pytest
-, mock
 , cmarkgfm
-, bleach
 , docutils
+, fetchPypi
 , future
+, mock
 , pygments
-, six
+, pytestCheckHook
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "readme_renderer";
-  version = "28.0";
+  version = "29.0";
+  disabled = pythonOlder "3.5";
```

```suggestion
  disabled = pythonOlder "3.6";
```
Support for 3.5 was dropped.

---

Ähh, damn copy-&-paste...